screening/modular-open-source-identity-plaform.json

```diff
@@ -0,0 +1,130 @@
+{
+  "name": "Modular Open Source Identity Plaform",
+  "ownership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://www.mosip.io/resource/ip-policy-trademark-and-copyright"
+  },
+  "dependencies": {
+    "mandatoryDepsCreateMoreRestrictions": "Yes",
+    "isSoftwarePltIndependent": "Yes",
+    "pltIndependenceDesc": "For evey closed source software and hardware component that MOSIP interacts with, we provide standard interfaces for interaction. Either by using existing open standards (ex: For HSM we use JCE), or by defining standard interfaces for conformance (ex: MOSIP biometrics interfaces)"
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "docs.mosip.io"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "nonPIIAccessMechanism": "The project allows each deployment to define what data they collect. Some of these could be non-PII data. "
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Unknown",
+    "privacyComplianceList": [
+      ""
+    ],
+    "isLocIntlAdherent": "Yes",
+    "locIntlComplianceList": [
```

Comment:
If they answer `Yes` to this question, then they must provide a list of laws they comply with

Reply:
Reached out to the contact person via email on 12th November, 2020 requesting for more information on the international and local compliance list. Shall update once they respond.

Reply:
MOSIP has now provided more information on the international and domestic compliance laws and can be found in this commit 0499368.